homeassistant/components/xiaomi_miio/strings.json

```
@@ -409,7 +409,7 @@
     },
     "light_night_light_mode_off": {
       "name": "Night light mode off",
```

to be consistent with the other names

```suggestion
      "name": "Light night light mode off",
```